Sources/XcodeGenKit/PBXProjGenerator.swift

```diff
@@ -391,9 +391,11 @@ public class PBXProjGenerator {
         var copyWatchReferences: [String] = []
         var extensions: [String] = []
         
-        let dependenciesAndTransient = getAllDependenciesPlusTransientNeedingEmbedding(target: target)
+        let transientlyLinkDependencies = target.transientlyLinkDependencies ?? project.options.transientlyLinkDependencies
+        let targetDependencies = (target.transientlyLinkDependencies ?? project.options.transientlyLinkDependencies) ?
```

Comment:
looks like this is the same as `transientlyLinkDependencies` above

Reply:
Opps, missed on refactor. Will remove.